build.gradle

```diff
@@ -485,6 +489,8 @@ def testTasksProvider = provider {
             testTasks << "${projectPath}:jacocoTestReport"
         }
     }
+
+    testTasks << "frontend:lintFrontend"
```

Comment:
What do you think about updating our lint GHA in CI.yml?  I don't like overlapping test cases with style/linting rules because it can prevent hide test failures from being executed and conversely makes it hard to identify where the style failures are run and reported.